docs/staticAnalysis.md

```diff
@@ -25,14 +25,23 @@ Conversely, when updating any tool, ensure that the tool version is supported by
 [CheckStyle](http://checkstyle.sourceforge.net/index.html) helps to enforce coding standard in Java source code.
 The rules to be used are configured in a ruleset file; in TEAMMATES the file can be found [here](../static-analysis/teammates-checkstyle.xml).
 The plugin for Eclipse can be found [here](http://eclipse-cs.sourceforge.net/#!/).
+The plugin for IntelliJ can be found [here](https://plugins.jetbrains.com/idea/plugin/1065-checkstyle-idea).
 
-#####Configuring Checkstyle Eclipse plugin  
+#####Configuring Checkstyle Eclipse plugin
 
 1. In `Project > Properties`, go to the `Checkstyle` tab.
 2. In the `Local Check Configurations tab`, create a new Check Configuration. Select `Project Relative Configuration` for its Type, enter any Name you wish and set the Location to the `teammates-checkstyle.xml` file in the Project Folder. Click OK.
 3. In the `Main` tab, uncheck `Use simple configuration`.
 4. Add a new File Set. It should include only the `.java$` file. Enter any name you wish for the `File Set Name`, and select the Check Configuration that you created earlier for `Check Configuration`. Click OK.
-5. Ensure that only the newly created File Set is enabled. Disable all other File Sets if they are enabled. Click OK. You have successfully setup the Checkstyle Eclipse plugin.
+5. Ensure that only the newly created File Set is enabled. Disable all other File Sets if they are enabled. Click OK.
+
+##### Configuring Checkstyle in IntelliJ IDEA
+
+1. Go to `File → Settings → Other Settings → Checkstyle`.
+1. Set `Scan Scope` to `Only Java sources (including tests)`.
+1. Click the `+` to add a new configuration file. Browse for `teammates-checkstyle.xml` in the `static-analysis` folder.
```

Comment:
@samsontmr 
Minor nit: `static-analysis` folder should be mentioned _before_ `teammates-checkstyle.xml`, to match the order in which users need those two items.
e.g. 
> Click the `Browse` button, navigate to the `static-analysis` folder, and choose the `teammates-checkstyle.xml` file.

May be create a first timer issue for this?